README.md

```diff
@@ -71,10 +71,13 @@ For working on integrating R2 into SimplyE, use the `feature/readium2` branch. F
 Then we recommend using the SimplifiedR2.workspace. This assumes you have checked out the following frameworks (clone them as siblings of `Simplified-iOS` on the file system):
 ```bash
 cd Simplified-iOS/..
-git clone https://github.com/readium/r2-shared-swift
-git clone https://github.com/readium/r2-streamer-swift
+git clone https://github.com/vladimirfedorov/r2-shared-swift
```

Comment:
This is temporary, this repositories add adobe DRM option to R2Shared and R2Streamer, because the original repositories are able to work with LCP only. As soon as Readium 2 gets multiple DRM support, we won't need these forked repositories.

Reply:
do we have an ETA for when R2 will add multiple DRM support? if we don't or if you think it will be longer than say 6 months (?) perhaps we should transfer these repos from your personal account to NYPL? Maybe @MalcolmMcFly  should chime in here for some direction, I don't know what are the conventions here.

Reply:
moving this to NYPL can be a good idea anyway

Reply:
we have https://github.com/NYPL-Simplified/r2-streamer-swift and https://github.com/NYPL-Simplified/r2-shared-swift now

Reply:
cool... do you know why they are not forks though? Is it possible to link them to the original repos, so we can easily integrate new updates done upstream? It is kind of essential especially because I believe R2 team is preparing a new release soon

Reply:
ha, indeed... Malcolm created them for me, I added these remotes and pushed, probably that's why. I'll check what can be done with that

Reply:
R2Streamer and R2Shared repositories in NYPL-Simplified are forks of Readium 2 repositories with a bit of my code now

Reply:
perfect, thanks! Is there anything that you submitted in the forks that you think we should submit upstream to the main R2 repos? Either now or some time in the future. Or are your commits strictly for SimplyE/NYPL usage?

Reply:
strictly for SimplyE/NYPL usage, nothing should be submitted upstream - the changes are only to make Adobe DRM work in our project